glean-core/python/glean/glean.py

```diff
@@ -132,56 +132,60 @@ def initialize(
 
         cls._upload_enabled = upload_enabled
```

Comment:
I just noticed this: should we call `set_upload_enabled()` instead? This would make it consistent with other platforms (can be a follow-up).

Reply:
Will you handle this in a followup?

Reply:
I just made this change right here.  Should be functionally equivalent, so low risk.